src/js/disability-benefits/containers/TurnInEvidencePage.jsx

```diff
@@ -54,12 +54,13 @@ class TurnInEvidencePage extends React.Component {
     if (this.props.loading) {
       content = <LoadingIndicator/>;
     } else {
+      const filesPath = `your-claims/${this.props.claim.id}/files`;
       content = (
         <div className="claim-container">
           <nav className="va-nav-breadcrumbs">
             <ul className="row va-nav-breadcrumbs-list" role="menubar" aria-label="Primary">
               <li><Link to="your-claims">Your claims</Link></li>
-              <li><Link to={`your-claims/${this.props.claim.id}`}>Your Compensation Claim</Link></li>
+              <li><Link to={filesPath}>Your Compensation Claim</Link></li>
```

Comment:
No, we're intentionally changing it to the files tab because that's really the parent of this page.